cmd/protoc-gen-gorums/gengorums/gorums.go

```diff
@@ -62,12 +65,20 @@ func GenerateFileContent(gen *protogen.Plugin, file *protogen.File, g *protogen.
 	g.P()
 	genGorumsMethods(data, gorumsCallTypes...)
 	g.P()
+	// generate all strict ordering methods
+	genStrictOrderingMethods(servicesData{g, file.Services}, strictOrderingCallTypes...)
+	g.P()
 }
 
 func genGorumsMethods(data servicesData, methodOptions ...*protoimpl.ExtensionInfo) {
 	g := data.GenFile
 	for _, service := range data.Services {
 		for _, method := range service.Methods {
+			// skip all strict ordering methods.
+			// they will be generated by genStrictOrderingMethods
+			if hasMethodOption(method, gorums.E_StrictOrdering) {
+				continue
+			}
```

Comment:
Together with other changes (to be suggested), the loop body can be replaced with this:
```go
	if hasMethodOption(method, gorums.E_StrictOrdering) {
		if hasStrictOrderingOption(method, methodOptions...) {
			fmt.Fprintf(os.Stderr, "processing %s\n", method.GoName)
			g.P(genGorumsMethod(g, method))
		}
	} else if hasMethodOption(method, methodOptions...) {
		fmt.Fprintf(os.Stderr, "processing %s\n", method.GoName)
		g.P(genGorumsMethod(g, method))
	}
```
(this will allow to eliminate `genStrictOrderingMethods`, which is very similar to `genGorumsMethods`...)